.github/workflows/format_php.yml

```diff
@@ -5,13 +5,17 @@ on:
     branches:
       - main
   pull_request:
+  pull_request_target:
```

Comment:
Hi @stefanzweifel, as discussed in https://github.com/stefanzweifel/git-auto-commit-action/issues/211, if you merge this PR and then open a new PR you should see that it works. Although I haven't tried having both `pull_request` and `pull_request_target` so not sure if there'll be conflict. Just including `pull_request_target` should be sufficient.

Reply:
Thanks @ConorMacBride will give this ago by mergin and making another PR. Back in my head I feared that this could be the issue. 😅